Documentation/gh-pages/source/introduction.rst

```diff
@@ -44,4 +46,24 @@ certain specific features in their own novel applications. Code contributions th
 can be submitted through GitHub and are always welcome.
 
 
-.. _KATRIN: https://www.katrin.kit.edu
+
+
+
+Supported operating systems and hardware requirements
+=====================================================
+
+*Kassiopeia* is supported and intended to run on systems running either Linux or MacOS X. Currently, it has been
+compiled and tested to run on fresh installations of the Linux distributions Fedora 31 and Ubuntu 20.04 LTS. It is also
+expected to compile and run on other Linux distributions, however this has not been tested, and the steps needed to
+compile *Kassiopeia* may deviate from what is outlined here.
+
+For minimal functionality and the ability to run the included example programs and simulations the following
+computer specifications or better are recommended:
+
+- Architecture: x86-64
+- CPU: Intel Core i3 @ 2.0 GHz
+- RAM: 4 GB
+- Free Disk Space: 10 GB
```

Comment:
```suggestion
```
Compilation nowadays is quite computationally intensive, I think this may well be very outdated.